x/mongo/driver/operation.go

```diff
@@ -46,6 +46,8 @@ var (
 	ErrReplyDocumentMismatch = errors.New("number of documents returned does not match numberReturned field")
 	// ErrNonPrimaryReadPref is returned when a read is attempted in a transaction with a non-primary read preference.
 	ErrNonPrimaryReadPref = errors.New("read preference in a transaction must be primary")
+	// ErrDatabaseNameEmpty occurs when database name is not provided
+	ErrDatabaseNameEmpty = errors.New("database name cannot be empty")
```

Comment:
We don't expect users to assert on this error directly, so it doesn't need to be exported.

```suggestion
	errDatabaseNameEmpty = errors.New("database name cannot be empty")
```